pyam/core.py

```diff
@@ -1284,6 +1287,48 @@ def export_meta(self, excel_writer, sheet_name='meta'):
         if close:
             excel_writer.close()
 
+    def to_datapackage(self, path, tmp='tmp'):
+        """Write object to a frictionless datapackage
+
+        More information: https://frictionlessdata.io
+
+        Returns the saved :class:`datapackage.Package`. When adding metadata
+        (descriptors), please follow the `template` defined by
+        https://github.com/OpenEnergyPlatform/metadata
+
+        Parameters
+        ----------
+        path: string
+            file path
+        tmp: string, default 'tmp'
+            name for temporary folder to export csv files
+            (deleted after saving datapackage)
+        """
+
+        path, tmp = Path(path), Path(tmp)
+        if tmp in list(tmp.parent.iterdir()):
+            msg = 'folder `{}` already exists in this directory!'
```

Comment:
If you still want to use `tmp`, then here you could just do `if tmp.exists():`.

Reply:
true...